.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -25,3 +29,6 @@ If applicable, add screenshots to help explain your problem.
 
 **Additional context**
 Add any other context about the problem here.
+
+- Does it happen all the time?
+- Did it use to work?
```

Comment:
```suggestion
- Did it used to work?
```

Reply:
or maybe something else?
```suggestion
- Did it work before?
```

Reply:
To my defense: I just copied the text from navcontainerhelper 😅